lib/GithubHandler.js

```diff
@@ -177,15 +177,15 @@ GithubHandler.prototype.pullRequestHandler = function(event, cb) {
     type: 'pull_request', // also in event.event
     service: 'github',
     host: undefined,
+    branch: event.payload.pull_request.head.ref,
```

Comment:
Sure, we can do that in the future. Having the full PR data, we can always backfill existing builds. But I'd wait until we have a concrete use case to update the build object directly in code now.